### ISSUE TITLE
add support for node-sass

### ISSUE THREAD

**Issue by TheAlphaNerd** [nodejs/citgm#165]
The tarball *should* have everything we need. We are getting some weird environmental failures though. Let's use this issue to track things as we get it working.

**Comment by xzyfer** [comment]
If there's a branch I'm happy to take a look

**Comment by TheAlphaNerd** [comment]
```
ENOENT: no such file or directory, scandir '/Users/thealphanerd/Downloads/node-sass-3.8.0/test/fixtures/spec/spec'
```

Looks like you also need to roll in the [test spec](https://github.com/sass/node-sass/blob/master/.gitmodules#L4) in the release tar ball as well so the test suite can run.

This can be reproduced in the [tar ball served from github](https://github.com/sass/node-sass/archive/v3.8.0.tar.gz) if you run `npm install && npm test`

[Here is a branch](https://github.com/TheAlphaNerd/citgm/blob/node-sass/) that includes `node-sass` [in the lookup](https://github.com/TheAlphaNerd/citgm/blob/node-sass/lib/lookup.json#L296-L299).

**Comment by xzyfer** [comment]
Ah I see. Will investigate if the size of sass-spec is prohibitive to publishing. If so we'll change `npm test` to not rely on sass-spec.

**Comment by TheAlphaNerd** [comment]
fwiw node-core ships all of our tests in the tarball, it's 17 of the 136 uncompressed mbs

**Comment by xzyfer** [comment]
Sass spec is part of a wider discussion for our team. We're using a relatively old version. Newer versions introduced a Ruby dependency so we need a plan for how to handle that.

Out of curiosity does citgm assume dist-tag latest or can you use a specific dist-tag?

**Comment by TheAlphaNerd** [comment]
@xzyfer right now it assumes `dist-tag` latest, but we also support "master". We could definitely do a sprint on supporting different `dist-tags` as well.

**Comment by xzyfer** [comment]
I could ship a POC dist-tag with the tests inlined until we've decided how to move forward with publishing tests. I expected to have a solution in the next 2 weeks or so incase that timeline matters.

**Comment by xzyfer** [comment]
Node Sass PR https://github.com/sass/node-sass/pull/1688

**Comment by TheAlphaNerd** [comment]
🎉

**Comment by xzyfer** [comment]
There is some additional discussion in the state of node-sass test suite that you may be interested, and/or helpful with - https://github.com/sass/node-sass/issues/1686

I've assumed the citgm doesn't CI doesn't guarantee the existence of Ruby.

**Comment by TheAlphaNerd** [comment]
it does not. the only thing we support is node + npm

**Comment by xzyfer** [comment]
Thanks for the clarification.

**Comment by xzyfer** [comment]
This should be resolved as of v3.9.0 \o/

**Comment by TheAlphaNerd** [comment]
fixed by 4b81864fcb
